android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/auth/ProxiedForConsentedUsersApi.java

```diff
@@ -0,0 +1,175 @@
+package org.sagebionetworks.bridge.android.manager.auth;
+
+import org.joda.time.DateTime;
+import org.joda.time.LocalDate;
+import org.sagebionetworks.bridge.rest.api.ForConsentedUsersApi;
+import org.sagebionetworks.bridge.rest.model.ConsentSignature;
+import org.sagebionetworks.bridge.rest.model.GuidHolder;
+import org.sagebionetworks.bridge.rest.model.Message;
+import org.sagebionetworks.bridge.rest.model.NotificationRegistration;
+import org.sagebionetworks.bridge.rest.model.NotificationRegistrationList;
+import org.sagebionetworks.bridge.rest.model.ReportDataList;
+import org.sagebionetworks.bridge.rest.model.ReportIndexList;
+import org.sagebionetworks.bridge.rest.model.ScheduleList;
+import org.sagebionetworks.bridge.rest.model.ScheduledActivity;
+import org.sagebionetworks.bridge.rest.model.ScheduledActivityList;
+import org.sagebionetworks.bridge.rest.model.StudyParticipant;
+import org.sagebionetworks.bridge.rest.model.Survey;
+import org.sagebionetworks.bridge.rest.model.UploadRequest;
+import org.sagebionetworks.bridge.rest.model.UploadSession;
+import org.sagebionetworks.bridge.rest.model.UploadValidationStatus;
+import org.sagebionetworks.bridge.rest.model.UserSessionInfo;
+import org.sagebionetworks.bridge.rest.model.Withdrawal;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+/**
+ * Proxies ForConsentedUserApi calls so it uses retrieves authenticated client from AuthenticationManager
```

Comment:
I think so too.  Seems there should be some way to inject the password into the ForConsentedUsersApi that Retrofit creates